### PR TITLE
Cleanup dummy app getting rid of unneeded code/configurations

### PR DESCRIPTION
**Pull request by tabfugnic**
Related issue: #9 

* Remove simple form
* Remove dependency on sass
* Remove stylesheets
* Remove images from assets
* Combine all configuration into test and remove development
* Remove a bunch of bin related things
* Fix setup to be more robust
* Remove spec support

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@
 /tmp/
 spec/examples.txt
 /spec/dummy/.byebug_history
+db/migrate/*_create_last_notifications.rb
```
Comment (wardpenney) [inline .gitignore:13]: What is that file?

**Comment by mike-burns** [comment]
Nice, love the diff stats.

I see some tests are removed in the process. Are those needed?

**Review by wardpenney — APPROVED** [review]
LGTM! One small question

**Comment by tabfugnic** [comment]
@mike-burns They were excess tests that I'm not convinced were testing the gem itself  but instead testing the rails application. Maybe some form of those should exist, but they should probably be moved into the main application somehow.

**Comment by tabfugnic** [comment]
They also mark a way the application is supposed to run that is a bit different than what is planned

**Comment by mike-burns** [comment]
> They also mark a way the application is supposed to run that is a bit different than what is planned

Ah, that makes sense. LGTM!

**Pull request merged**